### ISSUE TITLE
Touchpoints modal is not contained by a landmark in the app

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4002]
## Area or Component
Application-wide

## Quick Summary
The touchpoints modal is not part of the page's landmarks (i.e., `<header>, <footer>, or <main>`.) 

### How to Replicate
Inspect touchpoints modal in-app

## Accessibility Violation
All page content should be contained by landmarks: https://dequeuniversity.com/rules/axe/4.4/region?application=AxeChrome

## Action Requested
Add Touchpoints to the page landmarks. This might require us to create an application-wide footer.

## Original Deque Issue
  | region | Ensures all page content is contained by landmarks | Touchpoints modal not contained by landmark | All page content should be contained by landmarks | https://dequeuniversity.com/rules/axe/4.4/region?application=AxeChrome | moderate | No | No |   | [".touchpoints-wrapper"] | Fix any of the following:  Some page content is not contained by landmarks | <div class="touchpoints-wrapper sr-app-touchpoints-button">How can we improve SimpleReport?</div> | cat.keyboard,best-practice | rnv7@cdc.gov | Sign up selector | https://www.simplereport.gov/app/sign-up/ |   | 2022-07-15T19:09:47.072Z |   |   |   |   |   |   |   |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

**Comment by fzhao99** [comment]
We want to make sure that the fix for this issue doesn't run afoul of previous fixes to only have the one landmark on each page, as described [here](https://github.com/CDCgov/prime-simplereport/pull/4115#issuecomment-1201531514)

**Comment by emmastephenson** [comment]
@fzhao99 this may have already been fixed?
